.github/ISSUE_TEMPLATE/---releng-checklist.md

```diff
@@ -0,0 +1,53 @@
+---
+name: "✅ Releng checklist"
+about: Keep track of the release activities
+title: 'Releng checklist - vX.X'
+
+---
+
+##### Sprint Release Owner: _[name here]_
+
+#### Tuesday #1
+
+- [ ] Upgrade a-c
+
+#### Wednesday #1
+
+- [ ] Export strings (*if applicable*) ([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Localization#exporting-strings-for-translation))
+- [ ] Create GitHub pre-release build ([instructions](https://github.com/mozilla-mobile/firefox-tv/wiki/Releng-Checklist#final-builds))
```

Comment:
Do we need to do a version bump here too? If so, we should specify what the version naming should be